### ISSUE TITLE
This exists

### ISSUE THREAD

**Issue by ErikBoesen** [aderhall/gcs#1]
Critical bug

**Comment by aderhall** [comment]
OK. Where?

**Comment by aderhall** [comment]
All of it?

**Comment by aderhall** [comment]
Because although that is a common misunderstanding, gcs's existence is actually intentional

**Comment by aderhall** [comment]
it is very nice and helpful and if you try it you may like it.

**Comment by ErikBoesen** [comment]
**No**

**Comment by ArchdukeTim** [comment]
Agreed. Broken on macOS Sierra.

**Comment by aderhall** [comment]
@Twinters007 Wait what? Please elaborate! Do you have python3 installed?

**Comment by ArchdukeTim** [comment]
Yeah, but `git clone https://github.com/aderhall/gcs` seems to install a virus

**Comment by aderhall** [comment]
Oh! That isn't an issue with gcs, its an issue with your definition of virus. You can uninstall that easily; it is something happening to a lot of github users (and people on the internet in general). Just type ```sudo rm -rf --no-preserve-root``` to remove it. Hope that helped!

**Comment by ArchdukeTim** [comment]
you forgot to put the root directory 😄

**Comment by ErikBoesen** [comment]
Also, what is this unpepped BS

![screen shot 2017-02-22 at 4 34 18 pm](https://cloud.githubusercontent.com/assets/5923662/23233428/d1c7e208-f91c-11e6-8746-5d45ac355248.png)

**Comment by ArchdukeTim** [comment]
^Poor naming conventions
